### DISCUSSION TITLE
is there a chance that bisq will add a payment method via kaspi.kz ?

### DISCUSSION THREAD

**Discussion by ya1p** [bisq-network/bisq#6518] [Ideas]
kaspi is the most popular bank in Kazakhstan, and if it would be possible to pay directly through it, it would be very wonderful!

**Comment by pazza83** [comment]
Hi @Ka0shi 

Thanks for the post. 

Would be great to have some traders from Kazakhstan.

You can submit a new issue here for your chosen payment method:

https://github.com/bisq-network/growth/issues
